src/relation_engine_server/bulk_import.py

```diff
@@ -33,3 +35,17 @@ def _write_edge_key(json_line):
             json_line["_from"].encode() + json_line["_to"].encode(), digest_size=8
         ).hexdigest()
     return json_line
+
+
+def _import_from_file(file_path, query):
+    """Open a file of line-separated JSON and bulk-import it."""
+    with open(file_path, 'rb') as file_desc:
+        resp = requests.post(
+            db_url + '/_api/import',
+            data=file_desc,
+            auth=(db_user, db_pass),
+            params=query
+        )
+    if not resp.ok:
+        raise ArangoServerError(resp.text)
+    return resp.text
```

Comment:
I could make a case for why we'd want this function here or in arango_cilent (though I probably lean towards the latter) but if we do move it here, I think you should remove the bulk_import from arango_cilent